src/vellum/workflows/workflows/base.py

```diff
@@ -72,6 +72,7 @@
 from vellum.workflows.types.utils import get_original_base
 from vellum.workflows.utils.uuids import uuid4_from_hash
 from vellum.workflows.workflows.event_filters import workflow_event_filter
+from vellum_ee.workflows.display.workflows import BaseWorkflowDisplay
```

Comment:
[blocking] we cannot import from the `vellum_ee` into our the open source part of our library. Let's have this be a static method on the display class instead